projects/plugins/boost/app/assets/src/js/react-components/RegenerateCriticalCssSuggestion.tsx

```diff
@@ -0,0 +1,32 @@
+import { Notice } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { hideRegenerateCriticalCssSuggestion } from '../stores/config';
+
+export const RegenerateCriticalCssSuggestion = ( { show } ) => {
+	if ( ! show ) {
+		return null;
+	}
+
+	return (
+		<Notice
+			level="info"
+			title={ __( 'Regenerate Critical CSS', 'jetpack-boost' ) }
+			onClose={ () => {
+				hideRegenerateCriticalCssSuggestion();
+			} }
+		>
```

Comment:
IMHO we shouldn't show a close button on this if users can only temporarily dismiss it. When I click the X button I expect it not to come back when I refresh the page.

Reply:
Done